src/domains/dashboard/components/node-view/node-view.tsx

```diff
@@ -104,21 +107,22 @@ const SubSection = ({
             )),
         )}
       </div>
-      {submenuNames.map((submenu: string) => {
-        const submenuID = name2id(`menu_${menu}_submenu_${submenu}`)
-        const chartsSorted = menu.submenus[submenu].charts
+      {submenuNames.map((submenuName: string) => {
```

Comment:
Maybe we could separate this map function into different file?

Reply:
Sure, i can do that, do you mean function or separate component? But btw. don't you think it will a little complicate this, because you will have to open 2 files to actually read that and create map in head to comprehend it? The function will never be reused btw.